P-7155: Added SLA webhook doc (#48)
team-plain/docs: pages/ui-components.mdx

# UI Component

UI components are a way of describing some UI you want to appear in Plain.
They can be used when:

> - [creating a Thread](./threads)
> - [creating an Event](./events)
> - [in Customer Cards](/customer-cards)

Reviewer: ```suggestion
- [creating a Thread](/threads)
- [creating an Event](/events)
- [in Customer Cards](/customer-cards)
```

  ![Example button linking to stripe](/images/ui-components/link-button-stripe.png)
</ImageContainer>

> In the GraphQL API schema, we have three separate unions that all share contain the same types:

Reviewer: ```suggestion
In the GraphQL API schema, we have three separate unions that each contain the supported components for threads, events, and Customer Cards:
```

> - `CustomTimelineEntryComponent`
> - `CustomerCardComponent`
> - `EventComponent`

Reviewer: ```suggestion
- `CustomTimelineEntryComponent`
- `CustomerCardComponent`
- `EventComponent`

To see UI components in action you can experiment with them in the [UI Components playground](https://app.plain.com/developer/ui-components-playground/).
```

> - `EventComponent` - one exception here is that [Container](./ui-components/container) does not exist in `EventComponent`

Reviewer: ```suggestion
- `EventComponent`
```
